packages/server/src/MessageProcessor.js

```diff
@@ -48,6 +51,13 @@ type CachedDocumentType = {
   contents: Array<CachedContent>,
 };
 
+const KIND_TO_SYMBOL_KIND = {
+  Field: SymbolKind.Field,
+  OperationDefinition: SymbolKind.Class,
+  FragmentDefinition: SymbolKind.Class,
+  FragmentSpread: SymbolKind.Struct,
```

Comment:
Should we look to add more symbols in the future, e.g. `ObjectTypeExtension` and such?

Reply:
Sounds reasonable to me - I haven't worked very much with the SDL, so I only implemented what the outline code already does here: https://github.com/stephen/graphql-language-service/blob/90c4c5899f3f14216794636749210386fa749427/packages/interface/src/getOutline.js#L21-L30